Makefile

```diff
@@ -3,7 +3,7 @@ check: lint cs tests phpstan
 
 .PHONY: tests
 tests:
-	php vendor/bin/phpunit
+	php vendor/bin/phpunit --exclude-group=platform
```

Comment:
I'd think it'd make sense to do it the opposite way - when someone runs `vendor/bin/phpunit` do not run platform tests, only run them when someone specifically types `--group platform`. I do the same thing here: https://github.com/phpstan/phpstan-src/blob/1310ce93bf4ef94b1fdba1b90f22ccf003599623/phpunit.xml#L34-L39

Reply:
Adjusted.